entity/page.php

```diff
@@ -540,6 +544,9 @@ public function get_content_for_display($censor_text = true)
 	*/
 	public function set_content($content)
 	{
+		// Override editing time and maximum post characters limits
+		$this->config['edit_time'] = $this->config['max_post_chars'] = 0;
```

Comment:
`$this->config['edit_time']` has no effect here, it only applies in the posting.php script. What we are trying to defeat is the `form_token_lifetime` which by default has a 2 hour life time for editing any form.

This is easy to override by adding -1 to `check_form_key()`, in the admin_controller.php when we submit the form (line 228) like this:
```php
// Use -1 to allow unlimited time to submit form
if (!check_form_key('add_edit_page', -1))
```
@Wolfsblvt It's perfectly fine to overide the config value for max_post_chars since it only applies during the script execution of this set_content() function. It has no affect on anything else.